src/duckling/shell/shell.component.ts

```diff
@@ -18,16 +18,16 @@ import {StoreService} from '../state';
 
 @Component({
     selector: 'dk-shell',
-    styleUrls: ['./duckling/shell/shell.component.css'],
+    styleUrls: ['./duckling/shell/shell.component.css', './duckling/layout.css'],
```

Comment:
You should import layout into the shell.component.scss file instead of referencing it here